README.md

```diff
@@ -47,19 +47,20 @@ development environment
     - Con: Everything is running in containers. Need to use `docker container exec` to run things in container
 2. Follow the steps below to run Postgres in a docker container and the
 Dataservice on your local machine
-    - Pro: Run everything directly on local machine
+    - Pro: Maybe easier to develop and test, can use flask shell for debugging
```

Comment:
> Maybe easier to develop and test

Why is this option easier to develop and test? Because flask shell can be used for debugging? Or some other reason, e.g. because you don't:

> Need to use `docker container exec` to run things in container